wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,330 @@
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) | 2007-10-4 | October 2007 |  |
+| ![][flag_AU] [eyup](https://osu.ppy.sh/users/88) | October 2007 | 2007-10-11 |  |
+| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-10-11 | 2007-11-17 |  |
+| ![][flag_NZ] [Echo](https://osu.ppy.sh/users/431) | 2007-11-17 | 2007-11-23 |  |
+| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-11-23 | 2007-12-24 |  |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 2007-12-24 | *2008-09-07* |  |
+
+## 2008
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | *2007-12-24* | 2008-09-07 |  |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | 2008-09-07 | 2008-09-07 |  |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 2008-09-07 | 2008-10-07 |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2008-10-07 | 2008-12-11 |  |
+| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | 2008-12-11 | *2009-01-19* |  |
+
+## 2009
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | *2008-12-11* | 2009-01-19 |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-01-19 | 2009-03-05  |  |
+| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | 2009-03-05  | 2009-03-06  |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-03-06 | March 2009  |  |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | March 2009 | 2009-03-18 |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-03-18 | 2009-04-16 |  |
+| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | 2009-04-16 | 2009-04-18 |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-04-18 | June 2009 |  |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | June 2009 | July 2009 |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | July 2009 | 2009-07-05 |  |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 2009-07-05 | July 2009 |  |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | July 2009 | 2009-08-13 |  |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 2009-08-13 | 2009-09-03 |  |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | 2009-09-03 | October 2009 |  |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | October 2009 | November 2009 |  |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | November 2009 | *2010-02-07* |  |
+
+## 2010
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | *November 2009* | 2010-02-07 |  |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | 2010-02-07 | 2010-07-10 |  |
```

Comment:
```suggestion
| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | October 2009 | November 2009 |  |
| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 2009-12-08 | *2010-02-07* |  |

## 2010

| Player | From | To | Sources |
| --: | :-- | :-- | :-- |
| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | *2009-12-08* | 2010-01-03 |  |
| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | 2010-01-03 | 2010-07-10 |  |
```

by 8 December 2009 Doomsday achieves #1: https://i.imgur.com/8irCSvO.jpg

by 3 January 2011 Doomsday lost #1: https://i.imgur.com/X5MDhUp.jpg

Looking at the 7 February leaderboard, I think it's highly possible Ana_Coppola had #1 sometime during this period. They have sensible score for that, and that would also match up with them being an ex-rank 1 by 18 June. It could also explain the 2011 reign that seems nonexistent as confusion between 2011 and 2010.